pyproject.toml

```diff
@@ -138,6 +138,7 @@ mypy = [
 pytest-benchmark = ">=4.0.0"
 pytest-snapshot = ">=0.9.0"
 requests-mock = ">=1.10.0"
+rfc3339-validator = ">=0.1.4"
```

Comment:
This is apparently required to perform `date-time` validation with `python-jsonschema`. It might be good to document for target developers that they'd need to add this as a dependency if they want to perform `date-time` validation with `python-jsonschema`.

Reply:
I did notice that today.  This is seems simpler to me at least than adding the extras `format` and `format-nongpl` to the poetry install.   https://python-jsonschema.readthedocs.io/en/stable/#extras